.changeset/wicked-items-guess.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+`CodeBlock`: Fixed lineWrapping to make long strings wrap when they overflow the container
```

Comment:
We seem to be preferring [the dash notation](https://github.com/hashicorp/design-system/blob/main/wiki/Website-Changelog.md)
```suggestion
`CodeBlock`- Fixed `lineWrapping` to make long strings wrap when they overflow the container
```